metaflow/plugins/aip/aip_metaflow_step.py

```diff
@@ -398,6 +427,28 @@ def aip_metaflow_step(
         with open(output_file, "w") as f:
             f.write(str(values[idx]))
 
+    # get card and write to output file
+    card_cli: str = _get_cards_cli(
+        step_name,
+        task_id,
+        metaflow_run_id,
+        script_name,
+    )
+    cmd: str = card_cli.format(run_id=metaflow_run_id)
+    cmd = f"{cmd} > /tmp/outputs/cards/default_card.html"
```

Comment:
Does this mean that only default card is supported, but user can't add any other card yet?

Reply:
added comments no for it now, thanks!

Reply:
added an example to the repo of multiple card 

see:

https://metaflow.int.stage-k8s.zg-aip.net/NBFlow/argo-nbflow-lbnlj/end/end-3?direction=asc&group=true&order=startTime&section